app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,54 @@
+*{
+  padding: 0;
+  margin: 0;
+}
+
+#container {
+  margin: 48px auto;
+  width: 960px;
+
+  section {
+    clear: both;
+
+    p {
+      font-size: 22px;
+    }
+  }
+
+  header {
+    margin-bottom: 46px;
+
+    &.clearfix:after {
+      content: "";
```

Comment:
Properties should be ordered clear, content, display, height, line-height, visibility